src/language-js/print/array.js

```diff
@@ -23,6 +23,19 @@ import { printOptionalToken, printTypeAnnotation } from "./misc.js";
 
 /** @typedef {import("../../document/builders.js").Doc} Doc */
 
+function printEmptyArrayLike(path, openBracket, closeBracket, options) {
```

Comment:
Let's just call it `printEmptyArray`, other function ` printArray*` also not only work for array.

Reply:
[55b0e45](https://github.com/prettier/prettier/pull/13608/commits/55b0e45b43660db7f235d3cc6359796f372f1485)